modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -363,8 +363,8 @@ spec:
 <6> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
 <7> The administrative agent that routes the administrative requests to servers.
 <8> Set this value to `true` if you want to enable `nodeAgent` and perform File System Backup.
-<9> Enter `kopia` or `restic` as your uploader. You cannot change the selection after the installation. For the Built-in DataMover you must use Kopia. The `nodeAgent` deploys a daemon set, which means that the `nodeAgent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR.
-<10> Specify the nodes on which Kopia or Restic are available. By default, Kopia or Restic run on all nodes.
+<9> Enter `kopia` as your uploader, to use the Built-in DataMover. The `nodeAgent` deploys a daemon set, which means that the `nodeAgent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR.
```

Comment:
I see that this was approved so my questions probably aren't very valid, and they probably mostly come from a lack of understanding in how this YAML configuration works: 

* Does setting `kopia` as the uploader cause the `nodeAgent` to deploy a daemon set? 
*  defaultVolumesToFsBackup is set in the `Backup` CR. Should we call that out? eg:  

Comma splice

```suggestion
<9> Enter `kopia` as your uploader to use the Built-in DataMover. The `nodeAgent` deploys a daemon set, which means that the `nodeAgent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR. For more information, see "Backup custom resource". 
```

Maybe we don't call it out. That being said, at the very least the comma needs removed.